src/app_config.cpp

```diff
@@ -402,6 +419,25 @@ config_save_ohm(bool enable, String qohm)
   config.commit();
 }
 
+void
+config_save_current_shaper(bool enable, int maxpwr, String livepwr)
+{
+  uint32_t newflags = flags & ~CONFIG_SERVICE_CUR_SHAPER;
+  if(enable) {
+    newflags |= CONFIG_SERVICE_CUR_SHAPER;
+  }
+  config.set("flags", newflags);
+  config.set("mqtt_live_pwr", livepwr);
+  config.set("current_shaper_max_pwr", maxpwr);
+  config.commit();
+
+  // please fix this: I'm pretty sure it's not the way to do here, but I haven't got clear the subscription thing probably linked to this :)
+  // so forcing config_cha nged here:
+  config_changed("mqtt_live_pwr");
+  config_changed("current_shaper_max_pwr");
+
+}
+
```

Comment:
Should not create explicit functions, should just use the generic save

Reply:
Do you have any example somewhere ?

Reply:
All these save functions are lagacy and shouldn't be used for new stuff if you look at the JS they are mostly using the /config endpoint. There is a ticket to remove all of these. Sorry for the confusion.

Reply:
yep I saw that later, it's fixed now.